src/internal/version/backup.go

```diff
@@ -25,6 +25,11 @@ const (
 	// collection as the folder itself.
 	OneDrive4DirIncludesPermissions = 4
 
+	// OneDrive5DirMetaName changed the directory metadata file name from
+	// <dirname>.dirmeta to just .dirmeta to avoid issues with folder renames
+	// during incremental backups.
+	OneDrive5DirMetaName = 5
```

Comment:
Just a thought, how about `OneDrive5DirMetaNoName` to indicate that it is just an extension without a name?